src/librustc_typeck/check/method/suggest.rs

```diff
@@ -78,6 +78,30 @@ impl<'a, 'tcx> FnCtxt<'a, 'tcx> {
             return;
         }
 
+        let print_disambiguation_help = |
+            err: &mut DiagnosticBuilder<'_>,
+            trait_name: String,
+        | {
+            err.help(&format!("to disambiguate the method call, write `{}::{}({}{})` \
+                              instead",
+                              trait_name,
```

Comment:
Please change the formatting to be:

```
foo(&format!(
    "...",
    bar,
    baz,
));

Reply:
Cool 🙂 I actually did that but reverted it because I thought this was the preferred style. I'll adjust it accordingly.